### ISSUE TITLE
Scene summary should be located above control panel

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#114]
At the moment, the scene summary is located below the control panel due to the structure of the scene graph.  It would be optimal if the summary were located nearer the top of the document so that it is one of the first things found when using the cursor.

**Comment by jessegreenberg** [comment]
Looks great @jobara thanks! Commits above put description above the form.  Closing.
